static-assets/scripts/main.js

```diff
@@ -975,6 +988,11 @@
         no: formatMessage(words.no)
       };
 
+      var logoContainer = document.querySelector('#navbarLogo');
+      CrafterCMSNext.render(logoContainer, 'CrafterCMSFullLogo', {
+        className: 'crafter-studio-logo'
+      });
+
```

Comment:
Try rendering this through plain ReactDOM.render call to avoid the overhead of the whole Bridge for all these logo usages.

Reply:
Removed react rendering, using to images and showing the correct one with media queries